docs/learn/learn-async-backing.md

```diff
@@ -0,0 +1,146 @@
+   within that block.
+
+:::
+
+For more information on the validity and availability process, be sure to visit the
+[parachain protocol](../learn/learn-parachains-protocol.md) page.
+
+## Synchronous Backing on Polkadot
+
+With synchronous backing, there was only about a single relay chain block, or 6-second window, to
+complete the parablock inclusion process. This was tightly coupled to the relay chain's progress,
+where blocks had to be created within this window:
+
+```mermaid
+%%{init: { 'logLevel': 'debug', 'theme': 'neutral', 'themeVariables': { 'fontSize': '14px', 'commitLabelFontSize': '16px', 'tagLabelFontSize': '16px' }, 'gitGraph': {'showBranches': true, 'showCommitLabel':true,'mainBranchName': 'Relay Chain'}} }%%
+gitGraph
+commit id:"R1"
+branch "Parachain 1, Block 1" order: 2
+commit id:"Block 1 (Candidate)"
+commit id:"Block 1 (Backed)"
+checkout "Parachain 1, Block 1"
+commit id:"Block 1 (Included)"
+branch "Parachain 1, Block 2" order: 3
+checkout "Relay Chain"
+merge "Parachain 1, Block 1" tag:"P1 Included" id:"R2"
+checkout "Parachain 1, Block 2"
+commit id:"Block 2 (Candidate)"
+commit id:"Block 2 (Backed)"
+checkout "Parachain 1, Block 2"
+commit id:"Block 2 (Included)"
+checkout "Relay Chain"
+merge "Parachain 1, Block 2" tag:"P2 Included" id:"R3"
+```
+
+where **P1** and **P2** are two parablocks, and **R1** and **R2** are two relay-chain blocks. The
+letters within the parentheses represent the state of a parachain block. Hence **C** (candidate,
+created by collators), **B** (backed, by para-validators), and **I** (included, by block authors in
+the relay-chain). For more information about parablock validation see the
+[parachain protocol page](./learn-parachains-protocol.md).
+
+In the diagram there are three relay-chain blocks being finalized (**R1-2**) and a parachain
+(**Parachain 1**) proposing two parablocks (**P1** and **P2**). Each parablock's lifecycle, from
+being candidate (**P(C)**) to being backed and included (**P(B)** and **P(I)**, respectively), must
+fit within the relay-chain block lifetime.
+
+Thus, the main limitation of synchronous backing is that parablock validation is tightly coupled to
+the relay chain's progression on a 1-1 basis, meaning every parablock must be built, backed, and
+included within six seconds. Parablocks depend on being in sync with the relay chain, reducing the
+amount of data in the block.
+
+Essentially, a parablock is rushing to being validated by the relay-chain due to this synchrony.
+
+By making this process of backing parablocks more asynchronous, parachains get the chance to not
+only include more data within each parablock, but also retry to include parablocks that failed
+inclusion.
+
+## Asynchronous Backing on Polkadot
+
+With asynchronous backing, the window is more than the span of around two blocks, or a ~12-second
+window. This enables more computational and storage time per block, as the context of the next relay
```

Comment:
We aren't getting our execution context from the next relay block. It comes from the unincluded segment, which is a chain of recent ancestor parablocks. As the name suggests, these parablocks have yet to be included in a relay block.